app/views/donation_sites/create.js.erb

```diff
@@ -0,0 +1,6 @@
+<% flash.discard %>
```

Comment:
Using JS ERB is a really old pattern that we'd prefer not using. Turbo and Stimulus are more modern ways of achieving the same goal. Can you look into those approaches? We use them elsewhere in the app.